src/sentry/lang/native/processing.py

```diff
@@ -291,7 +291,7 @@ def _handles_frame(data, frame):
 
     # TODO: Consider ignoring platform
     platform = frame.get("platform") or data.get("platform")
-    return is_native_platform(platform) and "instruction_addr" in frame
+    return is_native_platform(platform) and frame.get("instruction_addr") is not None
```

Comment:
I'm sorry, can you clarify what the difference is between these two lines? Is it that the first one returns `true` if there is an `instruction_addr`, but it's equal to `None`?

Reply:
I think thats exactly what is going on, yes. Though I haven’t really tested this in detail, so meh 🤷🏻‍♂️

Reply:
This isn't exactly identical, but it sure rather obfuscated.  And without a unittest will regress.

From your other message the error reported is:
```
Failed to deserialize the JSON body into the target type: invalid type: null, expected a number or hex string at line 1 column 11380
```
So the problem is that some frames already contain `"instruction_addr": None` which passes the `"instruction_addr" in frame` test, but it would fail this one, because whether it is present or None the new code will return `True`.

Could we add a unittest?  Also, the schema actually wants an int or hex string.  Maybe we should validate the entire field here given this intention?